portfolio/src/main/webapp/index.html

```diff
@@ -7,30 +7,31 @@
 		<script src="script.js"></script>
 	</head>
 	<body>
-	<div id="content">
-		<h1>Jenny's Portfolio</h1>
+		<div id="content">
+			<h1>Jenny's Portfolio</h1>
 
-		<table>
-			<tr>
-				<th onclick=>about me</th>
-			</tr>
-			<tr>
-				<th>contact me</th>
-			</tr>
-		</table>
+			<table>
+				<th onclick="ToggleReset()">home</th>
+				<th onclick="ToggleAbout()">about me</th>
+				<th onclick="ToggleContact()">contact me</th>
+			</table>
 
-      
-    	<div id="MainPage">
+				
+			<div id="main-page">
```

Comment:
Nitpick: using a word other than "page" here (and the other similar places) would be more clear, as "page" is overloaded WRT web development. What about "main-section"?